tests/ui/test_home.py

```diff
@@ -21,3 +19,52 @@ def test_most_popular_extensions_are_sorted_by_users(
     extensions = page.most_popular.extensions
     sorted_by_users = sorted(extensions, key=lambda e: e.users, reverse=True)
     assert sorted_by_users == extensions
+
+
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_that_clicking_on_addon_name_loads_details_page(
+        my_base_url, selenium, initial_data):
+    page = Home(selenium, my_base_url).open()
+    page.click_on_first_addon()
+    page.wait_for_page
+    assert 'addon' in selenium.current_url
+
+
+@pytest.mark.smoke
+@pytest.mark.nondestructive
+def test_that_featured_themes_exist_on_the_home(
+        my_base_url, selenium, generate_themes):
+    page = Home(selenium, my_base_url).open()
+    assert u'Featured Themes See all \xbb', 'Featured Themes region title \
+        doesn\'t match' == page.featured_themes_title
+    assert page.featured_themes.count >= 6
+
+
+@pytest.mark.nondestructive
+def test_that_clicking_see_all_themes_link_works(
+        my_base_url, selenium, generate_themes):
+    page = Home(selenium, my_base_url).open()
+    page.featured_themes.click_see_all_link()
+    page.wait_for_page
+    assert 'themes' in selenium.current_url
```

Comment:
Here I would use a `Themes` page returned from `page.featured_themes.see_all()`. As mentioned before, this page object would include a suitable `wait_for_page_to_load`, and then you can check a detail from the page to gain confidence you're in the right place. If you want to use the URL then PyPOM helps by providing support for [URL templates](http://pypom.readthedocs.io/en/latest/user_guide.html#url-templates) and [seed URL](http://pypom.readthedocs.io/en/latest/user_guide.html#waiting-for-pages-to-load), but doesn't perform this implicitly because URLs can be rewritten after loading in some applications.